packages/gatsby-kontent-components/src/image-element/get-gatsby-image-data.ts

```diff
@@ -53,6 +53,8 @@ export function getGatsbyImageData({
     return `${baseUrl}?${query}`;
   };
 
+  urlBuilder({baseUrl:"www", width:200, height: 500, format: "", options})
```

Comment:
What is the purpose of this line?

Reply:
I just wanted to debug it, and somehow forget to remove it :/